stdlib/_curses.pyi

```diff
@@ -555,4 +555,8 @@ class _ncurses_version(NamedTuple):
     patch: int
 
 ncurses_version: _ncurses_version
-window = _CursesWindow  # undocumented
+
+# typeshed used the name _CursesWindow for the underlying C class before
+# it was mapped to the name 'window' in 3.8.
+# Kept here as a legacy alias in case any third-party code is relying on it.
+_CursesWindow = window
```

Comment:
I like the `curses._CursesWindow` alias, at least for now, but I think you should delete `_curses._CursesWindow`. If someone really imported `_CursesWindow` from `_curses`, they deserve to get new mypy errors :)